### ISSUE TITLE
node-entry.js:6 Uncaught TypeError: Cannot assign to read only property 'document' of object '#<Window>'

### ISSUE THREAD

**Issue by pjkarlik** [googlevr/webvr-polyfill#297]
using webpack and three.js I get this when adding as import into my js page..

node-entry.js:6 Uncaught TypeError: Cannot assign to read only property 'document' of object '#<Window>'

unsure if I'm importing or using it right in the context of webpack builds...

**Comment by jsantell** [comment]
This is for node-like environments where the globals do not exist, but something in that webpack configuration must be triggering this by providing a `global` object. Either way, we should check before overwriting these values. I'll have a PR shortly.

**Comment by jsantell** [comment]
Can you confirm if changing `src/node-entry` to this works for you?

```js
if (typeof global !== 'undefined' && global.window) {
  if (!global.document) {
    global.document = global.window.document;
  }
  if (!global.navigator) {
    global.navigator = global.window.navigator;
  }
}

require('./main');
```

**Comment by pjkarlik** [comment]
I no longer get that issue - that fixed the import issue using webpack!! YAY!!

**Comment by jsantell** [comment]
I just recreated this issue myself, thanks for reporting! I'll have a fix shortly

**Comment by jsantell** [comment]
Fixed via 878873d2edef3308f1d702abb72842c61541e8fe
